pkg/cmd/experimental/buildchain/test/multiple-namespaces-bcs.yaml

```diff
@@ -106,7 +106,7 @@ items:
           kind: DockerImage
           name: centos/ruby-22-centos7
         incremental: true
-        scripts: https://raw.githubusercontent.com/openshift/sti-ruby/master/2.0/.sti/bin
+        scripts: https://raw.githubusercontent.com/sclorg/s2i-ruby-container/master/2.2/s2i/bin
```

Comment:
i think we can just delete this, this should be the same scripts that are actually in the image, and the purpose of this test is not to check that the "scripts" field works (if it was, this test isn't working since as you noted, that url doesn't work anyway)